frontend/src/__tests__/cypress/cypress/pages/components/PipelinesTable.ts

```diff
@@ -0,0 +1,9 @@
+export class PipelinesTable {
+  private testId = 'pipelines-table';
+
+  find(): Cypress.Chainable<JQuery<HTMLElement>> {
+    return cy.findByTestId(this.testId);
+  }
+}
+
+export const pipelinesTable = new PipelinesTable();
```

Comment:
`pages/components` is meant to be a folder of generic UI utilities.
We already have a `pages/pipelines.ts` file for pipeline related page objects.

Reply:
Ok, I didn't realize we wanted to put multiple page objects in single files. I'll do this, but will that not make things a bit more difficult to navigate?

Reply:
Feel free to have separate files for different page objects. The main point here is that `components` was for generic page objects. Whereas specific ones should be in `pages/` or some other sub dir (which do not exist today).

Reply:
Ok, I'll keep that in mind when individual files seem to be getting too lengthy, I think creating sub-directories to organize pages and their supporting page objects is the way to go.